plugins/module_utils/healthchecksio.py

```diff
@@ -26,7 +26,7 @@ def json(self):
                 try:
                     json_body = json.loads(to_text(self.info["body"]))
                     return json_body
-                except json.decoder.JSONDecodeError:
+                except ValueError:
```

Comment:
`json.loads` will raise `ValueError` in Python2. In Python3 it raises `JSONDecodeError`. I suspect this is the reason for the warning as the github workflows are locked to Python 3.9.